### DISCUSSION TITLE
Getting started w/ Appium issue

### DISCUSSION THREAD

**Discussion by karahi** [webdriverio/webdriverio#7091] [Q&A]
Hi anyone, 

I started using webdriverio (with appium), and cant even get over the first hurdle - when I try to use a simple function in a spec-file like `browser.reload(...)`, `browser.timeouts(...)` or `browser.click(..)`, the process saying that those methods do not exist on the browser object.

Visual inspection (using back-to-basics `console.log()` ) confirms that no methods except "beforeX" and "afterX" hook arrays are defined on the `browser` (or the driver object). So....what stupid thing am i doing?

**Comment by christian-bromann** [comment]
> So....what stupid thing am i doing?

Can you share a bit of your code? Without it is difficult to make any assumptions.
